lc-hax/Scripts/Modules/ESPMod.cs

```diff
@@ -31,7 +31,7 @@ public class ESPMod : MonoBehaviour {
         if (!this.Enabled || !this.InGame || Helper.CurrentCamera is not Camera camera) return;
 
         this.PlayerRenderers.ForEach(rendererPair => {
-            if (rendererPair.GameObject.isPlayerDead || !rendererPair.GameObject.isPlayerControlled) return;
+            if (rendererPair.GameObject.isPlayerDead || !rendererPair.GameObject.isPlayerControlled || rendererPair.Renderer.bounds == null) return;
```

Comment:
Isn't bounds a struct and therefore, can't be null?

Reply:
well somehow in bepinex it kept spamming null exception.